Sources/Amplitude/Plugins/iOS/UIKitScreenViews.swift

```diff
@@ -0,0 +1,120 @@
+import Foundation
+#if os(iOS) || os(tvOS) || targetEnvironment(macCatalyst)
+import UIKit
+
+class UIKitScreenViews {
+    internal static let lock = NSLock()
+    internal static var amplitudes: [Amplitude] = []
```

Comment:
Should these be soft references? Possible memory leak?

Reply:
Fixed. Weak references now.